apps/console/src/index.jsp

```diff
@@ -153,6 +154,7 @@
     <script>
 
         var proxyContextPathGlobal = "<%= htmlWebpackPlugin.options.proxyContextPath %>";
+        var isSuperTenantRequiredInUrl = "<%= htmlWebpackPlugin.options.isSuperTenantRequiredInUrl %>";
```

Comment:
Do we need to add the default value to the index.html?

Reply:
added